### PR TITLE
[FIX] web, website: fix layout issue for Milk

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This PR fix two layout issues found in the current version of Milk : 

- Fixes the colors of the loader (eg. when you switch to dark mode)
- Fixes the "Edit" button of website in the upper right corner.

Requires : 
- https://github.com/odoo-dev/enterprise/pull/534

task-2818586

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
@stefanorigano 👋  here's the next fix for Milk, there's also a required PR in Enterprise.

File: addons/website/static/src/systray_items/edit_website.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .o_edit_website_container {
     &:not(.o_translatable) {
-        @include button-variant($o-brand-primary, $o-brand-primary);
+        margin: $o-navbar-padding-v * -1 0;
```
Comment (stefanorigano) [inline addons/website/static/src/systray_items/edit_website.scss]: Can we do this instead? I find it more readable 
```suggestion
        margin: -$o-navbar-padding-v 0;
```

Comment (Brieuc-brd) [reply addons/website/static/src/systray_items/edit_website.scss]: Fixed 👍

**Comment by stefanorigano** [comment]
@Brieuc-brd , cherry-picked, thanks 👍

**Pull request closed without merging**